tests/sqlparser_duckdb.rs

```diff
@@ -32,6 +32,39 @@ fn duckdb_and_generic() -> TestedDialects {
     }
 }
 
+#[test]
+fn test_struct() {
+    // basic struct
+    let canonical = r#"CREATE TABLE t1 (s STRUCT<v VARCHAR, i INTEGER>)"#;
+    let sql = r#"CREATE TABLE t1 (s STRUCT(v VARCHAR, i INTEGER))"#;
```

Comment:
Oh I think this is incorrect, no need for the input and output to not match, in this case the output is significantly different syntax from what went into the parser. We can probably add a `comma_delimeter: bool`  or similar flag to the StructField struct so that it knows how to display accordingly?

Reply:
I'm not clear about this. Do you mean add flag so we have different canonical name for `struct<>` and `struct()`?

Reply:
Ah yeah that's what I meant - my bad realised I wrote `comma_delimeter` but meant to write `parens_delimeter` instead. Yeah so like if that flag is true display uses `struct (...)` otherwise `struct<...>`